packages/react/accordion/src/Accordion.tsx

```diff
@@ -61,18 +67,23 @@ type AccordionSingleOwnProps = Polymorphic.Merge<
      */
     type: 'single';
```

Comment:
Can we remove these from here? This is the part I don't think you understood from my original feedback.

> I'm wondering if this should be moved up a level so it wouldn't even exist in these props [...]. It's only the parent that needs this prop

![image](https://user-images.githubusercontent.com/175330/118291483-91116b80-b4cf-11eb-99d9-b378732e74e9.png)


I was saying there that we would declare the `type` prop in the `AccordionOwnProps` and not in the lower level components that don't need it. Only the parent needs them to determine the switch.